lib/OpenQA/Command.pm

```diff
@@ -132,7 +133,8 @@ sub retry_tx ($self, $client, $tx, $retries = undef, $delay = undef) {
 "Request failed, hit error $res_code, retrying up to $retries more times after waiting … (delay: $delay; waited ${waited}s)\n"
         );
         sleep $delay;
-    }
+        $delay *= $factor;
+    } while ($retries--);
```

Comment:
Can we cover this in a unit test e.g. in `t/43-cli-api.t`?

And in case that's not clear. Your answer can be that it is not feasible or there is manual test steps that verify this better. Or maybe other reviewers don't find it necessary.

Reply:
a small test is added